app/models/concerns/account_interactions.rb

```diff
@@ -209,8 +209,6 @@ def lists_for_local_distribution
          .where('users.current_sign_in_at > ?', User::ACTIVE_DURATION.ago)
   end
 
-  private
```

Comment:
Why?

Reply:
ok. I was certain this is just a typo... why is the private just sitting there between these two defs?

I'll revert this change.